OmiseSDK/CardBrand.swift

```diff
@@ -35,7 +35,7 @@ import Foundation
         case .Visa:
             return "^4"
         case .MasterCard:
-            return "^5[1-5]"
+            return "^(5[1-5]|2(2(2[1-9]|[3-9])|[3-6]|7(0|1|20)))"
```

Comment:
How did you get this RegEx? Create by yourself?

Reply:
Founded from [Github](https://github.com/inacho/php-credit-card-validator/pull/9/files) repo but tested on https://regexr.com/ 🙇

Reply:
Cool, I was confused at first HAHAHA